### DISCUSSION TITLE
プロフィール画像の表示の高速化

### DISCUSSION THREAD

**Discussion by haruharu-1105** [aipictors/app#113] [アイデア - Ideas]
## 目的
通信制限環境化での、アプリの画像表示の高速化を行いたいです。

## 対応
計測していないので、憶測で申し訳ないのですが、プロフィール画像をキャッシュ表示して欲しいです。

https://github.com/aipictors/app/blob/3ab31ff8eb3a035dd8914336e47043a1b84502bb/lib/widgets/avatar/user_profile_avatar.dart#L27

https://github.com/aipictors/app/blob/3ab31ff8eb3a035dd8914336e47043a1b84502bb/lib/widgets/avatar/header_user_profile_avatar.dart#L29

https://github.com/aipictors/app/blob/3ab31ff8eb3a035dd8914336e47043a1b84502bb/lib/widgets/image/comment_sticker_image.dart#L18

## 懸念事項
ユーザー数が多いため、プロフィール画像をキャッシュすると、
キャッシュアルゴリズムによっては、投稿作品画像のヒット率が下がるかも！

## その他
Special Thanks!. Stella
https://docs.flutter.dev/perf/ui-performance

**Comment by Stella2211** [comment]
#114 で対応しました。
プロフィール画像だけでなく、通知や作品のポップアップ表示などにもキャッシュを利用するようにしました。

**Reply by haruharu-1105** [reply]
対応　ありがとうございました。
リリース後に確認します。
